.gitattributes

```diff
@@ -1,2 +1,4 @@
 *.png filter=lfs diff=lfs merge=lfs -text
 *.jpg filter=lfs diff=lfs merge=lfs -text
+
+api/** linguist-generated
```

Comment:
Mark all files in `api` as generated. This means GitHub will not show the diff by default.